index.js

```diff
@@ -128,11 +131,24 @@ module.exports = class PearUpdater extends ReadyResource {
       fork: this.drive.core.fork
     }
 
+    this.snapshot = this.drive.checkout(checkout.length)
+
+    try {
+      const latestPackage = JSON.parse(await this.snapshot.get('/package.json'))
+      const unskippableUpdates = (latestPackage.pear?.stage?.unskippableUpdates || [])
+        .filter(u => u > old.length)
```

Comment:
why is this under stage?

Reply:
also needs to check the key, decode it with hypercore-id-encoding and b4a compare against our key

Reply:
Moved it to pear.platform.unskippableUpdates. Is this alright or should it be something like pear.updates.unskippable?